apollo-router/src/plugins/telemetry/config_new/events.rs

```diff
@@ -803,31 +801,31 @@ mod tests {
 
         async {
             test_harness
-                .router_service(
-                    |_r|async  {
-                        Ok(router::Response::fake_builder()
-                            .header("custom-header", "val1")
-                            .header(CONTENT_LENGTH, "25")
-                            .header("x-log-request", HeaderValue::from_static("log"))
-                            .data(serde_json_bytes::json!({"data": "res"}))
-                            .build()
-                            .expect("expecting valid response"))
-                    },
-                )
+                .router_service(|_r| async {
+                    Ok(router::Response::fake_builder()
+                        .header("custom-header", "val1")
+                        .header(CONTENT_LENGTH, "25")
+                        .header("x-log-request", HeaderValue::from_static("log"))
+                        .data(serde_json_bytes::json!({"data": "res"}))
+                        .build()
+                        .expect("expecting valid response"))
+                })
```

Comment:
Snapshot for this test was changed, so I improved formating a bit during investigation